server/__init__.py

```diff
@@ -58,7 +58,7 @@ def create_app():
     app.register_blueprint(stats.bp)
 
     # Load chart config
-    with open('chart_config.json') as f:
+    with open('chart_config_new.json') as f:
```

Comment:
should we do an in-place replacement of the config?

Reply:
moved chart_config_new.json to chart_config.json